examples/001-dotnet-WebClient/Program.cs

```diff
@@ -23,7 +23,7 @@ public static class Program
     private static readonly List<string> s_toDelete = new();
 
     // Change this to True and configure Azure Document Intelligence to test OCR and support for images
-    private const bool imageSupportDemoEnabled = true;
```

Comment:
could you update the bicep template to include Azure Document Intelligence? that service is really useful and we're going to use it more. I rather leave this enabled for now.

Reply:
let me do this as a next PR

Reply:
let's leave this change out for now